command/debug_test.go

```diff
@@ -699,3 +689,83 @@ func TestDebugCommand_PartialPermissions(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestDebugCommand_MultiAddress(t *testing.T) {
+	t.Parallel()
+
+	conf, opts := teststorage.ClusterSetup(nil, nil, teststorage.InmemBackendSetup)
+	cluster := vault.NewTestCluster(t, conf, opts)
+	cluster.Start()
+	defer cluster.Cleanup()
+	testhelpers.WaitForActiveNodeAndStandbys(t, cluster)
+
+	var addrArgs []string
+	for _, core := range cluster.Cores {
+		addrArgs = append(addrArgs, "-addresses", core.Client.Address())
+	}
+	cases := []struct {
+		name string
+		args []string
+	}{
+		{
```

Comment:
not that critical, but you could add a test that does both addresses and cluster to validate that it errors

Reply:
Done